release_features.md

```diff
@@ -93,6 +99,9 @@ features, some of which may land in released versions of Gazebo.
 1. [Hydrodynamics for water surface vehicles.](https://github.com/gazebosim/gz-sim/pull/818)
 1. [Custom skybox from SDF.](https://github.com/gazebosim/sdformat/pull/1037)
 1. [Gz3D: support heightmaps, skybox and particles.](https://github.com/gazebo-web/gzweb/pull/35)
+1. [Tools for creating new Gazebo projects](https://github.com/gazebosim/gz_pkg_create)
+1. [Simplify determination of a sensor's topic name for rendering sensors.](https://github.com/gazebosim/gz-sim/pull/1908)
+1. [APIs that ease migration from classic.](https://github.com/gazebosim/gz-sim/issues/325)
```

Comment:
Checklist is done. Does the ticket need to be closed?

Reply:
Closed!